### PR TITLE
ansible: add a meter creating playbook

### PR DESCRIPTION
**Pull request by levex**
Signed-off-by: Levente Kurusa <levex@linux.com>

### CODE REVIEW

File: ansible/playbooks/create-meter.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# This playbook creates a new smart meter
+- hosts: all
+  remote_user: energy
+  become: yes
+  become_method: su
+  tasks:
+    - name: Update repositories
+      apt:
+        update_cache: yes
+    - name: Install dependencies for Docker
```
Comment (fangyi-zhou) [inline ansible/playbooks/create-meter.yaml]: no more docker

File: ansible/playbooks/create-meter.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        - software-properties-common
+        - python
+        - python-setuptools
+        - python-dev
+        - openssl
+        - libssl-dev
+        - libudev-dev
+        - daemon
+        - pwgen
+    - name: Install pip
```
Comment (fangyi-zhou) [inline ansible/playbooks/create-meter.yaml]: pip3

File: ansible/playbooks/create-meter.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        creates: /usr/bin/parity
+    - name: Create /energy directory
+      file: path=/energy state=directory
+    - name: Clone energyChain data
+      git:
+        repo: 'https://github.com/levex/group-project.git'
+        dest: /energy
+        update: yes
+        force: yes
+    - name: Run the meter
```
Comment (fangyi-zhou) [inline ansible/playbooks/create-meter.yaml]: Generates password

File: ansible/playbooks/create-meter.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      shell: curl -X POST -d `cat /energy/client_addr` http://146.169.47.73:6000/money
+      ignore_errors: yes
+    - name: write the account to config.toml
+      shell: echo "[account]" >> /energy/parity/res/config.toml && echo "  unlock = ['`cat /energy/client_addr`']" >> /energy/parity/res/config.toml && echo "  password = [\"/energy/client_pass\"]" >> /energy/parity/res/config.toml
+    - name: reload systemShit
+      shell: systemctl daemon-reload
+    - name: Enable ParityEnergy service
+      shell: systemctl enable /energy/parity/parity-energy.service
+    - name: Start ParityEnergy service
+      shell: systemctl start parity-energy
```
Comment (fangyi-zhou) [inline ansible/playbooks/create-meter.yaml]: How do you run rest of the system?

**Comment by fangyi-zhou** [comment]
Depends on other services. #84

**Comment by fangyi-zhou** [comment]
@danzvara please upload your version if yours are significantly different. Thanks

File: ansible/playbooks/create-meter.yaml (hunk 4 above)
Comment (levex) [reply ansible/playbooks/create-meter.yaml]: in the future we add the other services here, generally we should also use the systemd module of ansible, but it didn't work on the first few tries and i was tired

Comment (fangyi-zhou) [reply ansible/playbooks/create-meter.yaml]: #84 is to rescue

**Comment by fangyi-zhou** [comment]
Please review

**Comment by fangyi-zhou** [comment]
This will close #63

**Pull request merged**